### PR TITLE
imporve accessibility

### PR DESCRIPTION
**Pull request by AhmedMohsen600**
## What does this PR do
In this PR me and my coding partner check the error about the accessibility and make the changes.
## Description of the task to be completed
- [x] Checked the accessibility error
- [x] Added the label content and change the styles
- [x] Fixed the constraint color for the text
- [x] Added the alt attribute for the image
- [x] Added the heading in the correct order

## How should it be tested
Clone the repository
```
git clone https://github.com/AhmedMohsen600/portfolio-setup-mobile-first.git
```
Navigate to the folder
```
cd portfolio-setup-mobile-first
```
git checkout accessibility
```
#### Thanks for reviewing my PR 😊

### CODE REVIEW

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @AhmedMohsen600 ,
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
 :heavy_check_mark: Linters are passing.
:heavy_check_mark: You have a good code structure.
:heavy_check_mark: Your linters are correct.
 
 ## Required Changes 
 _Check the comments under the review._
 
 ## Optional suggestions 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -17,7 +17,11 @@
   <body>
     <header>
       <nav class="header">
-        <img class="logo" src="./imgs/logo.png" alt="logo" />
+        <img
+          class="logo"
+          src="./imgs/logo.png"
+          alt="this is the logo of my webpage"
+        />
```
[the remark this answers is not in the document]
Comment (AhmedMohsen600) [reply index.html:24]: @Alfredbis29 Thank you for your suggestion but do you mean to add tabindex for each element on HTML file  ??

[the remark this answers is not in the document]
Comment (AhmedMohsen600) [reply index.html:24]: @Alfredbis29 Thank you for your suggestion but I really don't know what to do

[the remark this answers is not in the document]
Comment (AhmedMohsen600) [reply index.html:24]: @Alfredbis29 If I have like 50 headings in my project should I add 50 tabindex for each one of them?? 
Doesn't make sense for me!

**Review by karlavdelgadof — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi there team :technologist: :wave: :sunglasses: ! 

You have successfully completed this project! There is nothing else to say other than... it's time to merge it! 🚀
Time to celebrate, you have done a fantastic job, congratulations! 🎉



## :sparkles: Highlights :sparkles:

- The accessibility percentage is 98%  :heavy_check_mark: :rocket: :+1: 
- Great work using the right GitHub Flow :100: 
- Awesome file organization :card_file_box: 
- Commits are descriptive, good work. 
- Linters are working properly, great job!

APPROVED! 💯 💯 💯

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

- [ ] **[OPTIONAL]:** For your `.btn-skills` I would suggest checking the contrast ratio, in this case, you could try using a darker shade for the text :+1: 

image


Cheers and best of luck with the following projects! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me **@karlavdelgadof**  in your question so I can receive the notification.
  
  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
  
  ------
  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**